.github/CODEOWNERS

```diff
@@ -47,4 +47,12 @@ spec/support/*/facilities @department-of-veterans-affairs/vsa-facilities-backend
 spec/support/*/lighthouse @department-of-veterans-affairs/vsa-facilities-backend @department-of-veterans-affairs/backend-review-group
 
 # test user dashboard product
-modules/test_user_dashboard @department-of-veterans-affairs/vsp-identity
```

Comment:
Did the trailing newline get removed here?

Reply:
I see "no newline at the end of file"

Reply:
I don't think the original had a new line at the end of the file, to begin with. I turned off that feature on my editor and saved and committed it again without this new line.

Reply:
Maybe the CODEOWNERS file is setup to be handled in this fashion?